ceph_deploy/hosts/debian/install.py

```diff
@@ -30,11 +30,11 @@ def install(distro, version_kind, version, adjust_repos, **kw):
         distro.packager.add_repo_gpg_key(gpg.url(key, protocol=protocol))
 
         if version_kind == 'stable':
-            url = 'http://download.ceph.com/debian-{version}/'.format(
+            url = 'https://download.ceph.com/debian-{version}/'.format(
```

Comment:
I might need to see if the SSL cert in use is valid "out of the box" on wheezy.  Right above this code is a special case where we fallback to http on wheeze to access to the GPG key, because otherwise we get invalid cert errors.